manifests/server/match_block.pp

```diff
@@ -8,8 +8,9 @@
   String $type   = 'user',
   Integer $order = 50,
   Stdlib::Absolutepath $target = $ssh::params::sshd_config,
-) {
-  include ssh
+) inherits ssh::params {
```

Comment:
Can you call inherits on a define() these days? I assumed it is for classes only

Reply:
Oops. This is definitely invalid and we should use a different method. I've fixed this in the current change.